ast/Trees.h

```diff
@@ -1147,9 +1252,30 @@ EXPRESSION(ConstantLit) {
     std::string nodeName() const;
     std::optional<std::pair<core::SymbolRef, std::vector<core::NameRef>>> fullUnresolvedPath(core::Context ctx) const;
 
+    core::SymbolRef symbol() const {
+        return resolutionScopesOrSymbol.symbol();
+    }
+
+    void setSymbol(core::SymbolRef symbol) {
+        resolutionScopesOrSymbol = symbol;
+    }
```

Comment:
What should the behavior be if you try to `setSymbol(StubModule())`?

Reply:
Good call.  I think it is correct to `ENFORCE` that we should not be doing this; I added a commit.